docs/src/integrate/nodejs-api.md

```diff
@@ -133,6 +133,8 @@ The `ESLint` constructor takes an `options` object. If you omit the `options` ob
   Default is `null`. Ignore file patterns to use in addition to config ignores. These patterns are relative to `cwd`.
 * `options.passOnNoPatterns` (`boolean`)<br>
   Default is `false`. When set to `true`, missing patterns cause the linting operation to short circuit and not report any failures.
+* `options.stats` (`boolean`)<br>
+  Default is `false`. When set to `true`, additional statistics are added to the lint results (see [Stats type](../extend/stats#-stats-type)).
```

Comment:
Can you move this into the Linting section below, as this option isn't related to File Enumeration?